pkg/kubectl/describe.go

```diff
@@ -953,6 +960,56 @@ func describeNode(node *api.Node, pods []*api.Pod, events *api.EventList) (strin
 	})
 }
 
+// HorizontalPodAutoscalerDescriber generates information about a horizontal pod autoscaler.
```

Comment:
Adding a test right now would require a FakeExperimentalClient to be created (lots of boilerplate code). I will do it but let's have it in a separate PR.